### DISCUSSION TITLE
Localizing pagination text

### DISCUSSION THREAD

**Discussion by Yslot** [Megabit/Blazorise#2132] [Q&A] [answered]
Is there a way to localize item pagination text? 
![Screenshot_9](https://user-images.githubusercontent.com/15019691/113028457-9323a300-918b-11eb-938c-889c7156bc10.png)

**Comment by stsrki** [comment, marked as answer]
See this https://github.com/stsrki/Blazorise/issues/2128

**Reply by Yslot** [reply]
Thanks. Works like a charm.
